frontend/public/components/dashboard/dashboard-card/card-help.tsx

```diff
@@ -2,20 +2,21 @@ import * as React from 'react';
 import { Button, OverlayTrigger, Popover } from 'patternfly-react';
 import { InfoCircleIcon } from '@patternfly/react-icons';
 
-export const DashboardCardHelp: React.FC<DashboardCardTitleHelpProps> = React.memo(({ children }) => {
+export const DashboardCardHelp: React.FC<DashboardCardTitleHelpProps> = React.memo(({ children, iconClassName }) => {
   if (React.Children.count(children) === 0) {
     return null;
   }
   const overlay = <Popover id="popover">{children}</Popover>;
   return (
     <OverlayTrigger overlay={overlay} placement="top" trigger={['click']} rootClose>
       <Button bsStyle="link">
-        <InfoCircleIcon className="co-dashboard-header__icon" />
+        <InfoCircleIcon className={iconClassName ? 'nb-object-dashboard-header__icon':'co-dashboard-header__icon'} />
```

Comment:
This is not right, you dont want to include `nb-object-dashboard-header__icon` class directly to core code. 
I guess you wanted to be able to specify additional classes ? like `classNames('co-dashboard-header__icon', iconClassName)` ?

Reply:
ack

Reply:
@a2batic why do we need to change the info icon size, we can have all icon of same sizes for all dashboards.

Reply:
@afreen23 they were looking comparatively big with the small text, so thought of changing. But we can keep them same.

Reply:
yes @a2batic we can keep them same to be consistent with overall dashboard